src/esp/gfx/Drawable.h

```diff
@@ -46,6 +48,9 @@ class Drawable : public Magnum::SceneGraph::Drawable3D {
 
   virtual void setLightSetup(const Magnum::ResourceKey& lightSetup){};
 
+  virtual Magnum::ResourceKey getOriginalMaterial() { return Magnum::ResourceKey{}; }
```

Comment:
These are needed to support the "find-and-replace" logic in `overrideMaterialForSubTree`.